### DISCUSSION TITLE
able to disable oklch and use hsl or something

### DISCUSSION THREAD

**Discussion by LeCyreaxYT** [saadeghi/daisyui#3101] [Ideas / feature requests]
I currently started a project on a game with a web render and the problem is that the current chromium version the oklch not supporting and the fallback vars not working

So is there any chance to get a disable function.

**Comment by saadeghi** [comment]
I would suggest using daisyUI 3.x until you can update the chromium because daisyUI uses other new CSS features anyway and with an old version of chromium we can't promise things to work as expected.

Let me know if you have any questions
